src/leapYearCalculator/dosite-leap_year_calculator.js

```diff
@@ -0,0 +1,26 @@
+function leapYearChecker(year) {
+    // Write your code here
+    var year;
```

Comment:
I believe that this line is not needed. You did not use the variable declared here anywhere.

Reply:
Remove that line and push again